snapcraft/internal/parser.py

```diff
@@ -205,7 +205,11 @@ def _process_entry(data):
 
     try:
         origin_data = _get_origin_data(origin_dir)
-    except (BadSnapcraftYAMLError, MissingSnapcraftYAMLError) as e:
+    except MissingSnapcraftYAMLError:
+        raise InvalidWikiEntryError(
+            'Origin {origin!r} is missing a snapcraft.yaml file'.format(
```

Comment:
Please finish the sentence with a period.
I know that we are really inconsistent about this, but I'd like to start adding periods everywhere.

Other than that, +1.